pkg/backends/newrelic/newrelic.go

```diff
@@ -303,10 +328,23 @@ func NewClientFromViper(v *viper.Viper) (gostatsd.Backend, error) {
 	nr.SetDefault("enable-http2", defaultEnableHttp2)
 	nr.SetDefault("user-agent", defaultUserAgent)
 
+	//New Relic Config Defaults & Recommendations
+	v.SetDefault("statser-type", "null")
+	v.SetDefault("flush-interval", "10s")
+	if v.GetDuration("flush-interval").Seconds() < 10 {
+		log.Warnf("[%s] flush-interval below 10s, recommended to be >= 10s", BackendName)
+	} else if v.GetDuration("flush-interval").Seconds() == 10 {
+		log.Infof("[%s] flush-interval default 10s", BackendName)
```

Comment:
It doesn't make a lot of sense to only log this when it's 10s, I'd just make it an "else" not an "else if".  That being said, I  suggest pushing this down to line 393.

Reply:
Shuffled and switched to else.